$OpenBSD$

Use OpenBSD-style default configuration and log directories for xftp-server.

--- apps/xftp-server/Main.hs.orig
+++ apps/xftp-server/Main.hs
@@ -9,6 +9,6 @@
 defaultCfgPath :: FilePath
-defaultCfgPath = "/etc/opt/simplex-xftp"
+defaultCfgPath = "/etc/simplex/xftp-server"
 
 defaultLogPath :: FilePath
-defaultLogPath = "/var/opt/simplex-xftp"
+defaultLogPath = "/var/simplex/xftp-server"
 
